spring-cloud-starter-stream-source-sftp/src/main/resources/META-INF/spring-configuration-metadata-whitelist.properties

```diff
@@ -1,4 +1,5 @@
 configuration-properties.classes=org.springframework.cloud.stream.app.sftp.source.SftpSourceProperties, \
+  org.springframework.cloud.stream.app.sftp.source.batch.SftpSourceBatchProperties, \
```

Comment:
Looks like you also have there an `SftpSourceRedisIdempotentReceiverProperties`

Reply:
updated